r/tests/testthat/test-compute-arith.R

```diff
@@ -133,3 +133,38 @@ test_that("Dates casting", {
   # Error: NotImplemented: Function add_checked has no kernel matching input types (array[date32[day]], scalar[double])
   expect_equal(a + 2, Array$create(c((Sys.Date() + 1:4) + 2), NA_integer_))
 })
+
+test_that("Math group generics work on Array objects", {
+  expect_equal(abs(Array$create(c(-1L, 1L))), Array$create(c(1L, 1L)))
+  expect_equal(
+    sign(Array$create(c(-1L, 1L))),
```

Comment:
Just to make clear that it's not the identity function

```suggestion
    sign(Array$create(c(-5L, 2L))),
```